apps/explorer/src/utils/envUtil.ts

```diff
@@ -8,4 +8,4 @@ export const DEFAULT_NETWORK =
     (import.meta.env.DEV ? Network.LOCAL : Network.DEVNET);
 
 export const IS_STATIC_ENV = DEFAULT_NETWORK === Network.STATIC;
-export const IS_STAGING_ENV = DEFAULT_NETWORK === Network.STAGING;
+export const IS_STAGING_ENV = DEFAULT_NETWORK === Network.LOCAL;
```

Comment:
Is this intentional?

Reply:
Yeah, because there will be no more staging env for explorer.

Reply:
@Jordan-Mysten FYI, this does not work well with pagination.